.devcontainer/Dockerfile.base

```diff
@@ -0,0 +1,42 @@
+FROM runpod/pytorch:2.4.0-py3.11-cuda12.4.1-devel-ubuntu22.04
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    wget \
+    nano \
+    socat \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Miniconda (optional, if you want it ready at build-time)
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh \
+   && bash /tmp/miniconda.sh -b -p /miniconda3 \
+   && rm /tmp/miniconda.sh
+
+ENV PATH="/miniconda3/bin:${PATH}"
+RUN eval "$(/miniconda3/bin/conda shell.bash hook)"
+RUN conda create -n comfyui python=3.11 -y
+RUN conda create -n comfystream python=3.11 -y
+COPY . /app
```

Comment:
The builds can be much faster if you first copy only the `requirements.txt` files and do the corresponding `pip install`. e.g. check these 2 copies: https://github.com/livepeer/ai-worker/blob/364ce77b58158ed088462a5a42ecbcf121bc94d4/runner/docker/Dockerfile.live-app__PIPELINE__#L14-L24

This will allow docker engine to cache the dependencies installation, which don't change that often, and rebuilds when you change only source files on comfystream will be pretty much immediate